lib/tasks/nebraska_rates20210813.rake

```diff
@@ -0,0 +1,725 @@
+# frozen_string_literal: true
+
+# This will enter all NE Hourly & Daily Rates for Family Child Care Home I & II as of 2021/08/13, subject to change
+desc 'Import all Nebraska Hourly and Daily Rates for Family Child Care Home I & II as of 2021/08/13'
+task nebraska_rates20210813: :environment do
```

Comment:
This imports all the rates as far as I interpreted them, and could use extra eyes cc @JonErvin Note that these are only for family child care home licenses, not for centers